contracts/src/mocks/IProxyAdminForBinding.sol

```diff
@@ -0,0 +1,9 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/nitro/blob/master/LICENSE
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+contract ProxyAdminForBinding is ProxyAdmin {}
```

Comment:
This was my mistake but let's rename this file to `ProxyAdminForBinding.sol` (I originally had it as an interface)

Reply:
done.